numpy/core/meson.build

```diff
@@ -79,6 +79,10 @@ endif
 # important, because code in NumPy typically does not check the value but only
 # whether the symbol is defined. So `#define HAVE_SOMETHING 0` is wrong.
 
+#cmake = import('cmake')
+#hwy = cmake.subproject('highway')
```

Comment:
@Mousius was this useful for local debugging? I'm working on Meson subproject usage elsewhere, so I'm curious about the details here.